yt/utilities/parallel_tools/tests/test_dask_helper.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+import pytest
+
+from yt.config import ytcfg
+from yt.testing import assert_array_equal, assert_equal
+from yt.utilities.on_demand_imports import _dask as dask
+from yt.utilities.parallel_tools import dask_helper
+
+
+@pytest.fixture(autouse=True, scope="module")
+def with_dask_enabled():
+    # Fixture to set dask_enabled config to standard value then reset.
+    # ensures that these tests will work even if default value for dask_enabled
+    # changes while ensuring the config value gets reset to the default value
+    # after a test runs.
+
+    original_config = ytcfg.get("yt", "internals", "dask_enabled")
+    ytcfg.set("yt", "internals", "dask_enabled", False)
+
+    yield  # test will run here
+
+    ytcfg.set("yt", "internals", "dask_enabled", original_config)
+
+
+@pytest.mark.skipif(not dask.__is_available__, reason="requires dask")
+def test_compute_helper(dask_client_fixture):
+    ytcfg.set("yt", "internals", "dask_enabled", True)  # make sure its enabled
+    # ensure that results returned by dask_helper.compute match those
+    # returned by standard dask compute methods.
+    f = dask.array.random.random((1000, 5), chunks=500)
```

Comment:
shouldn't matter here since it's comparing the result of an operation on the array between a standard method (`dask.compute`) and the wrapped method -- `f` is the same for both methods.